### PR TITLE
Doc: Glossary bulk upload

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Updating docs for glossary bulk upload

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/215404720-5c38167c-2cf8-4b63-a990-6e6cf547ca0a.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by chirag-madlani — APPROVED** [review]

File: openmetadata-docs/content/openmetadata/glossaries/index.md [hunk 1]
```diff
@@ -24,4 +24,39 @@ The terms from the glossary can be used for labeling or tagging as additional me
...
 alt="Glossary page demo"
-/>  
\ No newline at end of file
+/>
+
+## Glossary Bulk upload
+
+With glossary bulk upload you can create/update multiple glossary terms simultaneously. Below are steps for the same.
+
+- Create glossary in which you want to upload glossary terms `csvEg. Business Glossary`.
```
Comment (ShilpaVernekar) [inline openmetadata-docs/content/openmetadata/glossaries/index.md:33]: Create `a` glossary
Plz add space between csv & Eg.

File: openmetadata-docs/content/openmetadata/glossaries/index.md [hunk 2]
```diff
@@ -24,4 +24,39 @@ The terms from the glossary can be used for labeling or tagging as additional me
...
+- Click on 3 dots and click on export button (if you have terms already you will get CSV file containing all the terms, if you don't have any terms you will get blank CSV template).
```
Comment (ShilpaVernekar) [inline openmetadata-docs/content/openmetadata/glossaries/index.md:34]: Plz replace with this:
Click on the 3 dots icon and click on the Export button. If you have glossary terms in your Glossary, the same will be exported as a CSV file. If there are no terms in the Glossary currently, then a blank CSV template will be downloaded.

**Review by ShilpaVernekar — COMMENTED** [review]
Hi @ShaileshParmar11 

As discussed, the changes have been updated in the [doc](https://docs.google.com/document/d/13eKRZMbSGErhIFnR6ve3zxW7rL01jXbhcqoXcxx6DkQ/edit)

**Review by ShilpaVernekar — CHANGES_REQUESTED** [review]
Changes in mentioned doc

**Review by ShilpaVernekar — APPROVED** [review]
Looks good

**Pull request merged**